index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/30e1c676d3.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <main>
+      <header>
+        <strong>Welcome</strong>
+        <i class="fa-solid fa-bars"></i>
+      </header>
+      <section id="welcome">
+        <h1 id="welcome-heading">
+          <span id="heading-secondary">Hey there. I’m Bahir</span>
+          <br />
+          <span id="heading-primary"> I’m a software developer</span>
+        </h1>
+        <p id="description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div id="social-icons">
+          <div class="social-icon">
+            <i class="fa-brands fa-facebook"></i>
+          </div>
+          <div class="social-icon">
+            <i class="fa-brands fa-twitter"></i>
+          </div>
+          <div class="social-icon">
+            <i class="fa-brands fa-linkedin"></i>
+          </div>
+          <div class="social-icon">
+            <i class="fa-brands fa-whatsapp"></i>
+          </div>
+          <div class="social-icon">
+            <i class="fa-solid fa-envelope"></i>
+          </div>
+        </div>
```

Comment:
- [ ] I would kindly suggest the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 👍 👍